### ISSUE TITLE
Icons shift in dark/light mode

### ISSUE THREAD

**Issue by vinniefalco** [cppalliance/temp-site#503]
The search, help, and light/dark mode icons shift horizontally when changing between light and dark mode:
![image](https://github.com/cppalliance/temp-site/assets/1503976/3b8997fb-385a-4264-85e2-82b5cda575ff)

**Comment by gregnewman** [comment]
After testing and inspecting code this is being caused by that dark mode scrollbar that someone added.

**Comment by vinniefalco** [comment]
The scrollbars for light and dark should be have identical dimensions in every respect

**Comment by gregnewman** [comment]
@vinniefalco they don't.  It looks like Donovan (from blame) only created styles for the dark mode and he's setting some svg images on it so it's causing a jump.

**Comment by vinniefalco** [comment]
Yep that work was wasted, we need to throw it out and do it over (the scrollbars not the whole site).
